includes/class-newspack-popups.php

```diff
@@ -537,6 +538,18 @@ public static function register_meta() {
 			]
 		);
 
+		\register_meta(
```

Comment:
Do we also need to register `deactivation_date`?

Reply:
yes, done